README.md

```diff
@@ -114,49 +114,105 @@ The Vonage Video API Reference App for React is currently supported on the lates
 
 ## Running Locally
 
-### Config
+### 1. Ensure You Have a Vonage Account
 
- In project directory, create the environment variables for the project.
+To get started, make sure you have a Vonage account. You can create one at the [Vonage API Dashboard](https://dashboard.vonage.com/applications). The sign-up process is straightforward.
 
-```console
-cp backend/.env.example backend/.env
-```
-[Click here](backend/.env.example) to learn more about config variables used in the backend.
+### 2. Create an Application in the Dashboard
 
-Add your Vonage Video API credentials to the newly created .env file.
+Once logged in, navigate to the [Applications page](https://dashboard.vonage.com/applications) via the main dashboard menu:
 
-```console
-cp frontend/.env.example frontend/.env
+<details open>
+<summary>Applications dashboard view</summary>
+<img src="./docs/assets/readme/1-dashboard-applications.png" alt="Applications dashboard" style="max-width: 100%; height: auto;" />
+</details>
+
+If you don’t already have an application, create a new one:
+
+<details open>
+<summary>Create new app</summary>
+<img src="./docs/assets/readme/2-create-app.png" alt="Create app button" style="max-width: 100%; height: auto;" />
+</details>
+
+During the setup process, make sure to:
+
+- Provide a name for your application.
+- Generate and download the public and private keys.
+- Enable **Video** capabilities.
+
+Refer to the following image for visual guidance:
+
+<details open>
+<summary>Configuring a new app</summary>
+<img src="./docs/assets/readme/3-create-app-form.png" alt="Create app form" style="max-width: 100%; height: auto;" />
+</details>
+
+### 3. Environment Variable
+
+In the root project directory, create the environment files by running:
+
+``` bash
+cp backend/.env.example backend/.env && cp frontend/.env.example frontend/.env
 ```
-[Click here](frontend/.env.example) to learn more about config variables used in the frontend.
 
-### Running the project
+Then, open **backend/.env** and fill in the required configuration:
 
-#### Installing dependencies
+- **VONAGE_APP_ID** – This is the ID of your Vonage application. You can find it on the [Applications page](https://dashboard.vonage.com/applications).
+- **VONAGE_PRIVATE_KEY** – If you've already generated a private key, use that. Otherwise, use the key you downloaded when creating the app.
 
-```console
+### 4. Running the Project
+
+#### Install Dependencies
+
+``` bash
 yarn
 ```
-This command installs all appropriate dependencies for the project. If you would like more information on the packages we use, please refer to the [Dependencies](./docs/DEPENDENCIES.md) document.
 
-#### Dev mode
+This command installs all necessary dependencies. For details about the packages used, refer to [Dependencies](./docs/DEPENDENCIES.md).
```

Comment:
The way it was worded before was fine - I don't think there is a need to change it.